Frontend/ui-driver/src/Resource/Localizable/EN.purs

```diff
@@ -1288,7 +1288,7 @@ getEN stringKey =
         UNABLE_TO_LOAD_QUIZ_PLEASE_TRY_AGAIN -> "Unable to load quiz. Please try again"
         WE_GUARANTEE_YOU -> "We guarantee you:"
         LOWEST_FEES_FROM -> "Lowest Fees from {}"
-        ZERO_FEE_TILL -> "ZERO Fee till Oct 1st 2024"
+        ZERO_FEE_TILL -> "ZERO Fee till {}"
```

Comment:
why we doing in this way ? we can pass date string as argument like `ZERO_FEE_TILL date -> "ZERO Fee till date"`